apps/demos/Demos/Stepper/StepTemplate/Angular/app/app.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="stepper-label">Custom Step Shape</div>
+<dx-stepper
```

Comment:
please don't forget to tune a11y in this or another PR. I believe there should be `labelledby` linked to title